packages/plugin/vite/src/util/plugins.ts

```diff
@@ -33,3 +35,48 @@ export function externalBuiltins() {
     },
   };
 }
+
+/**
+ * Hot restart App during development for better DX.
+ */
+export function hotRestart(
+  config: VitePluginBuildConfig,
+  // For `VitePluginBuildConfig['restart']` callback args.
+  context: { renderers: RendererConfigWithServer[] }
+) {
+  const restart = () => {
+    // https://github.com/electron/forge/blob/v6.1.1/packages/api/core/src/api/start.ts#L204-L211
+    process.stdin.emit('data', 'rs');
```

Comment:
@MarshallOfSound pointed out to me that using `stdin` here is a bit of a hack here and there are edge cases where `process.stdin` could be null.

Ideally, we should instead expose additional start logic from the `@electron-forge/core` API to expose this functionality so that plugins can hook into it.

Reply:
Okay! This does seem a little dangerous. 🚨